src/actions.js

```diff
@@ -0,0 +1,14 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const pokemonApi = createApi({
+  reducerPath: "pokemonApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "https://pokeapi.co/api/v2/" }),
+  endpoints: (builder) => ({
+    getPokemonByName: builder.query({
+      query: () => `pokemon/?limit=20`,
```

Comment:
pass limit as parameter so that it can be changed. (limit = 20) => `pokemon/?limit=${limit}`